src/pacer.js

```diff
@@ -236,14 +237,16 @@ let PACER = {
           }
         }
         // xxx does not match style above.
```

Comment:
This `xxx` comment is no longer apposite.

Reply:
Removing comment.